include/conduit/conduit.h

```diff
@@ -147,7 +147,7 @@ namespace detail
     }
 
     template <typename T>
-    typename std::enable_if<!detail::has_putto_operator<T>::value>::type print_arg(std::ostream &stream, T &&t)
+    typename std::enable_if<!detail::has_putto_operator<T>::value>::type print_arg(std::ostream &stream, __attribute__ ((unused)) T &&t)
```

Comment:
In c++ you can get the same effect by just ommitting the variable name, so `T &&`.